src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -575,5 +577,7 @@ void uct_rc_mlx5_iface_commom_clean_srq(uct_rc_mlx5_iface_common_t *mlx5_common_
     }
 
     mlx5_cq->cq_ci             += nfreed;
-    rc_iface->rx.srq.available += nfreed;
+    if (rc_iface) {
```

Comment:
what is the use-case when we destroy CQ/SRQ, but do not destroy iface?

Reply:
The use case - we reset QP, and clean CQ - which can be either RX case (than we have to update SRQ too) or TX (no SRQ)

Reply:
Ok, then maybe add assert that srq is not NULL here? Since we expect both rc_iface and SRQ to be not NULL together